project/Versions.scala

```diff
@@ -0,0 +1,125 @@
+package akka
+
+import sbt._
+import Keys._
+
+object Versions {
```

Comment:
* If we want, we can link this in docs for referencing versions if needed
* Makes it very clean and fast to upgrade versions
* Almost all version-related logic is now here, where makes sense

Reply:
When I want to check dependencies and versions of an unfamiliar project I look for `Dependencies.scala`. Should we stick to the convention of having these in Dependencies.scala?

For docs we can use https://github.com/lightbend/sbt-paradox-dependencies, which was developed and used by Alpakka team.

Reply:
* WRT convention, I've found more projects handle versions for dependencies in a Version(s) object. In Akka we have to name it Versions. 
* Having the configuration of dependencies mixed in with the versions is not as fast for upgrades and not as clean 

Generally I find it messy and better to compartmentalize this way.

Reply:
ok